iuriMorari/src/main/java/Address.java

```diff
@@ -0,0 +1,25 @@
+
+public class Address {
+    public String country;
+    public String city;
+    public String street;
+
+    public Address(String country, String city, String street) {
+        this.country = country;
+        this.city = city;
+        this.street = street;
+    }
+
+    public String getFullAddress() {
+        return "Address is " + this.city + "" + this.country + "" + this.street;
```

Comment:
as another option you can use join() method provided within String class in order to connect two strings. (Optional)